dali/test/python/test_operator_reduce.py

```diff
@@ -257,3 +257,77 @@ def test_reduce_with_mean_input():
                 for type_id in types:
                     for ddof in [0, 1]:
                         yield run_reduce, keep_dims, reduce_fns, batch_gen, type_id, None, True, ddof
+
+
+def run_and_compare_with_layout(batch_fn, pipe):
```

Comment:
```suggestion
def run_and_compare_with_layout(batch_gen, pipe):
```
batch_fn is a bit misleading

Reply:
Done

Reply:
not done?

Reply:
Done, just wasn't pushed